app_customers/lib/pages/dashboard/dashboard_view.dart

```diff
@@ -136,62 +140,132 @@ class _DashboardBody extends StatelessWidget {
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     final localizations = Get.find<AppInternationalization>();
+
+    /// Variable for drawer control.
+    final customDrawerController = Get.find<CustomDrawerController>();
     final controller = context.watch<DashboardController>();
 
     return Scaffold(
       resizeToAvoidBottomInset: false,
       extendBody: true,
-      bottomNavigationBar: AnimatedBottomNavigationBar.builder(
-        itemCount: pages.length,
-        tabBuilder: ((index, isActive) {
-          final color = isActive
-              ? theme.bottomNavigationBarTheme.selectedItemColor
-              : theme.bottomNavigationBarTheme.unselectedItemColor;
-
-          return Column(
-            mainAxisSize: MainAxisSize.min,
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              Icon(
-                pages[index].iconData,
-                size: 24,
-                color: color,
-              ),
-              const SizedBox(height: 4),
-              Padding(
-                padding: const EdgeInsets.symmetric(horizontal: 8),
-                child: AutoSizeText(
-                  pages[index].title,
-                  maxLines: 1,
-                  style: TextStyle(color: color),
-                  group: autoSizeGroup,
+      bottomNavigationBar: Obx(
+        () => TweenAnimationBuilder(
+          tween: Tween<double>(
+            begin: 0,
+            end: customDrawerController.value.toDouble(),
```

Comment:
Instead of converting everywhere the type of `value` to double,  change the type to double.